wagtail_extensions/mixins.py

```diff
@@ -13,7 +13,7 @@
 class ContactMixin(models.Model):
     form_class = ContactForm
     success_url = None
-    success_message = 'Thank you! We will get back to you as soon as possible.'
+    success_message = ''
```

Comment:
I think I'd rather keep the default message here; so by default the form shows a success message.